config/analytics_hidden_pii.yml

```diff
@@ -1,13 +1,16 @@
 ---
 shared:
-  :registration_interests:
-    - email
-  :private_childcare_providers:
-    - registered_person_name
+  :applications:
+    - on_submission_trn
   :contracts:
-    - id
```

Comment:
removed after talking to @t-phillips-dfe - as hiding primary keys can cause problems.